test/model/content_test.dart

```diff
@@ -1412,13 +1412,43 @@ void main() {
     testParse('explicit paragraphs',
       // "* a\n\n  b"
       '<ul>\n<li>\n<p>a</p>\n<p>b</p>\n</li>\n</ul>', const [
-        ListNode(ListStyle.unordered, [
+        UnorderedListNode(items: [
           [
             ParagraphNode(links: null, nodes: [TextNode('a')]),
             ParagraphNode(links: null, nodes: [TextNode('b')]),
           ],
         ]),
       ]);
+
+    testParse('ordered list - large start number (9999)',
+      // "9999. first\n10000. second"
+      '<ol start="9999">\n<li>first</li>\n<li>second</li>\n</ol>', const [
```

Comment:
Let's put these new test cases in the new-style pattern that we use for most of the tests in this file: add new ContentExample instances as statics on ContentExample.

For a guide, see the comment at the top of that class.